src/tools/merssh/command.cpp

```diff
@@ -223,11 +226,31 @@ bool Command::isValid() const
     return true;
 }
 
+static QString readRelativeRoot()
+{
+    QString result;
+    FileReader reader;
+    if (reader.fetch("CMakeCache.txt")) {
+        QString data = QString::fromUtf8(reader.data());
+        QRegularExpression cmakeHomeRegexp("CMAKE_HOME_DIRECTORY:INTERNAL=(.*)");
+        QRegularExpressionMatch cmakeHomeMatch = cmakeHomeRegexp.match(data);
+        if (cmakeHomeMatch.hasMatch()) {
+            QString cmakeHome = cmakeHomeMatch.captured(1);
+            result = cmakeHome + '/' + QDir(cmakeHome).relativeFilePath("/");
+        }
+    }
+    return result;
+}
+
 void Command::maybeDoCMakePathMapping()
 {
     if (!QFile::exists("CMakeCache.txt"))
         return;
 
+    QString sharedTargetRoot = sharedTargetPath() + "/" + targetName();
+    QString relativeRoot = readRelativeRoot();
+    Q_ASSERT(!relativeRoot.isEmpty());
```

Comment:
Inactive in release build, use QTC_CHECK please.

Reply:
fixed